tests/Agent/IntegrationTests/IntegrationTests/RemoteServiceFixtures/MockNewRelicFixture.cs

```diff
@@ -17,12 +17,13 @@ public class MockNewRelicFixture : RemoteApplicationFixture
     {
         private const string ApplicationDirectoryName = @"MockNewRelic";
         private const string ExecutableName = @"MockNewRelic.exe";
+        private const string TargetFramework = "net6";
 
         public RemoteService MockNewRelicApplication { get; set; }
 
         public MockNewRelicFixture(RemoteApplication remoteApplication) : base(remoteApplication)
         {
-            MockNewRelicApplication = new RemoteService(ApplicationDirectoryName, ExecutableName, ApplicationType.Bounded, true, true, true);
+            MockNewRelicApplication = new RemoteService(ApplicationDirectoryName, ExecutableName, TargetFramework, ApplicationType.Bounded, true, true, true); ; ;
```

Comment:
Woah there turbo, I see you like semicolons! :)

Reply:
👍